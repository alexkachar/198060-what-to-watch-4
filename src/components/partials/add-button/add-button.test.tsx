import * as React from 'react';
import * as renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom';

import AddButton from './add-button';

it(`Footer renders correctly`, () => {
  const tree = renderer
    .create(
        <BrowserRouter>
          <AddButton
            id={1}
            isAuth={true}
            isFavorite={false}
            onSetFavoriteStatus={jest.fn()} />
        </BrowserRouter>, {
          createNodeMock: () => document.createElement(`div`)
        })
    .toJSON();
  expect(tree).toMatchSnapshot();
});
