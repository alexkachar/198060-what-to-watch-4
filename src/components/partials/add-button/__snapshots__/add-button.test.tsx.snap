// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddButton renders correctly 1`] = `
<a
  className="btn btn--list movie-card__button"
  href="/"
  onClick={[Function]}
>
  <svg
    height={20}
    viewBox="0 0 19 20"
    width={19}
  >
    <use
      xlinkHref="#add"
    />
  </svg>
  <span>
    My list
  </span>
</a>
`;
